### PR TITLE
Changed: Form Grid block styles

### LINKED ISSUE #6406: Overriding the "Donate" button text for the Form Grid on a per form basis should work
When I try to override the "Donate" button text on a per form basis, it's not working as of 2.20.0-beta-1
Steps:
1. Create a form.
1. on that form's edit screen, go to "Form Grid" tab.
1. Select "customize"
1. Modify the Donate button text.
1. Save the form.
1. On a new page, create a Form Grid
1. Note that the button text for the form you just created is not customized on that form.

https://givewp.canny.io/admin/board/release-testing/p/overriding-the-donate-button-text-for-the-form-grid-on-a-per-form-basis-should-w

### PR DESCRIPTION
**Pull request by alaca**
## Description

This PR updates the styles of the Form Grid block to better match the design in [Figma](https://www.figma.com/file/mF8Hn0p6Oc3yIQ2WpHHHH0/Form-Grid-Updated-Designs?node-id=42%3A648).

Also, the issue with overriding the Donate button text is resolved. Closes #6406.

Additional updates:

- Progress bar values now work correctly
- New Inspector Control layout
- Goals based on donor amount, donation amount and percentages display in the progress bar section
- Tags display on form grid in two alternate ways depending on featured images

## Affects

Form Grid block layout

## Visuals

https://user-images.githubusercontent.com/75056371/173704700-30c31bcc-ec53-4f2d-9cbb-cdd8b41e864d.mov

## Testing Instructions

-Create several forms with different donation goal options set.
-Process Donations on all forms
-Add excerpt messages, tags and featured images to some or all the forms
-View the Form Grid block
-Test all inspector controls to verify they work properly.
-Verify new design against the figma details

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [x] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit and/or end-to-end tests
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: assets/src/css/frontend/form-grid.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.give-form-grid-card__body {
+    align-items: center;
```
Comment (LMNTL) [inline assets/src/css/frontend/form-grid.scss:2]: The position of the donate button is kind of inconsistent depending the contents of the form:

image

Maybe adding `justify-content: space-evenly` or something like that would help?

**Review by LMNTL — CHANGES_REQUESTED** [review]
This looks great! Just one minor positioning thing to fiddle with.

**Comment by JasonTheAdams** [comment]
@alaca Honestly, all that really needs to be done on this PR is to change the button. The rest of the work will be done by Joshua. We just need the button looking nicer for 2.20.0.

File: assets/src/css/frontend/form-grid.scss (hunk 1 above)
Comment (alaca) [reply assets/src/css/frontend/form-grid.scss:2]: Thanks for the suggestion, implemented 48538b10 
That helped :raised_hands:

**Comment by alaca** [comment]
@JasonTheAdams there is still room for additional style improvements which can be done by Joshua.

**Comment by DevinWalker** [comment]
Thanks for your effort on this @alaca  - I agree this needs some more polish before going live and I think @JoshuaHungDinh can help us there along with Jeffrey's help to give it that extra refined look we're going for.

**Comment by jonwaldstein** [comment]
@JoshuaHungDinh my grid does not look like yours 😟 .  The full width was the only one really working properly, once I started using columns everything looks off.  Also it got worse when I limited the amount of forms.  I wonder if some styles got overridden during merge? Are you experiencing any of this?

Screen Shot 2022-06-10 at 11 51 42 AM

Screen Shot 2022-06-10 at 11 51 15 AM

**Review by JasonTheAdams — APPROVED** [review]
I hopped on a call and took a look with @JoshuaHungDinh. It worked perfectly fine on my machine. We made a couple small tweaks but it otherwise looked great!

**Review by rickalday — APPROVED** [review]

**Pull request merged**